src/core/data-cite.js

```diff
@@ -66,45 +66,40 @@ function cleanElement(elem) {
     .forEach(attrName => elem.removeAttribute(attrName));
 }
 
+function makeComponentFinder(component) {
```

Comment:
move it to utils?

Reply:
It's a bit specific for the micro syntax in data-cite. Might leave it here for now.